packages/lisk-api-client/package.json

```diff
@@ -28,6 +28,7 @@
 		"browserify:browsertest": "browserify ./browsertest.build/test/*.js ./browsertest.build/test/**/*.js -o ./browsertest.build/browsertest.js -s lisk",
 		"uglify": "uglifyjs -nm -o ./dist-browser/index.min.js ./dist-browser/index.js",
 		"uglify:browsertest": "uglifyjs -o ./browsertest.build/browsertest.min.js ./browsertest.build/browsertest.js",
+		"clean": "./scripts/cleanup.sh",
```

Comment:
Why not just use the same name for the script?

Reply:
Should we stick with clean or cleanup ? @willclarktech ?

Reply:
I have a mild preference for `clean`